README.md

```diff
@@ -36,13 +36,13 @@ This is possible because [we use the Parcel library](https://en.parceljs.org/scs
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_calendar/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_calendar/report/html_report/)
+- [DEMO LINK](https://aleks-pashkevych.github.io/layout_calendar/)
+- [TEST REPORT LINK](https://aleks-pashkevych.github.io/layout_calendar/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="978" alt="image" src="https://github.com/user-attachments/assets/0578a2fd-c5a5-41ae-af55-d82e9f29ba5f" />